WPMediaPicker.podspec

```diff
@@ -16,7 +16,8 @@ Pod::Spec.new do |s|
   s.source           = { :git => "https://github.com/wordpress-mobile/MediaPicker-iOS.git", :tag => s.version.to_s }
 
   s.platform     = :ios, '11.0'
-  s.requires_arc = true 
+  s.requires_arc = true
```

Comment:
Wonder if this can just be removed since the default is `true`
https://guides.cocoapods.org/syntax/podspec.html#requires_arc

Reply:
That's interesting! TIL. I didn't add this parameter, it already existed. I think I fixed the whitespace on it. What do you think about adding a task that reviews all of our .podspecs and makes sure they're all consistent?

Reply:
> What do you think about adding a task that reviews all of our .podspecs and makes sure they're all consistent?

👍 That could make sense. At least 10 dependencies seem to have the flag and I did notice some inconsistency with the Readmes which might be nice to unify just to make it easier to identify the current version.